Alicerce/Sources/Logging/Log+FileProvider.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  Log+FileProvider.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 07/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+public extension Log {
+
+    public class FileProvider: LogProvider {
+
+        public var minLevel: Log.Level = .error
+        public var formatter: LogItemFormatter = Log.ItemStringFormatter()
```

Comment:
While generally makes sense, in this particular case ConsoleProviderwouldn't conform to the LogProviderprotocol anymore.